modules/module-headings.php

```diff
@@ -12,213 +12,213 @@ function jetpack_get_module_i18n( $key ) {
 	static $modules;
 	if ( ! isset( $modules ) ) {
 		$modules = array(
-			'carousel'              => array(
```

Comment:
Looks like it was committed by mistake. can you revert these changes?

Reply:
Merging master into this branch reverted the changes